samples/net/zperf/src/zperf_shell.c

```diff
@@ -1105,13 +1145,13 @@ SHELL_CREATE_STATIC_SUBCMD_SET(zperf_cmd_tcp)
 							"(with suffix K)\n"
 		  "Example: tcp upload2 v6 1 1K\n"
 		  "Example: tcp upload2 v4\n"
-#if defined(CONFIG_NET_IPV6) && defined(MY_IP6ADDR_SET)
-		  "Default IPv6 address is " MY_IP6ADDR
-		  ", destination [" DST_IP6ADDR "]:" DEF_PORT_STR "\n"
+#if defined(CONFIG_NET_IPV6) && defined(MY_STATIC_IP6ADDR_SET)
```

Comment:
Doing `defined(MY_STATIC_IP6ADDR_SET)` will probably not work as expected as the value is always defined (it is either a const string or NULL).